src/controllers.lisp

```diff
@@ -16,11 +16,13 @@
 (defmethod on-click (instance x y))
 (defmethod on-middle-click (instance x y))
 (defmethod on-right-click (instance x y))
+(defmethod on-click-release (instance x y))
+(defmethod on-middle-click-release (instance x y))
```

Comment:
would it make sense to add `-press` methods as well?

Reply:
Hmm, I was intuitively interpreting `on-click` to mean "on press". But now that you say it, onclick events in JavaScript are triggered when the click is released. So it might make more sense that way.